report.md

```diff
@@ -686,6 +686,12 @@ Assigning values to address state variables without checking for `address(0)`.
 	    function foo2(address bar) public pure {
 	```
 
+- Found in src/cloc/HeavilyCommentedContract.sol [Line: 31](tests/contract-playground/src/cloc/HeavilyCommentedContract.sol#L31)
```

Comment:
@alexroan  I just added an `id()` method on ASTNode... How come this is newly added to the report? (it's not like I changed a solidity file...) 👀

Reply:
UPDATE: (i found out)

Please run `forge build --ast` in tests/cotract-playground before running `cli/reportgen.sh`.. You will see that these 2 were missing